addons/website_profile/static/src/js/website_profile.js

```diff
@@ -0,0 +1,56 @@
+odoo.define('website_profile.website_profile', function (require) {
+'use strict';
+
+var sAnimations = require('website.content.snippets.animation');
+
+sAnimations.registry.websiteProfile = sAnimations.Class.extend({
+    selector: '.o_wprofile_email_validation_container',
+    read_events: {
+        'click .send_validation_email': '_onSendValidationEmailClick',
+        'click .validated_email_close': '_onCloseValidatedEmailClick',
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+    _get_validation_email_params: function ($element) {
```

Comment:
this will become useless IMHO if we are doing the unique 'redirect_to' param

Reply:
Your Humble Opinion is marvelous !